### PR TITLE
Create v1.0.1.md

### PR DESCRIPTION
**Pull request by rebeccazzzz**
Release note draft for v1.0.1

### CODE REVIEW

**Review by yasker — DISMISSED** [review]

File: v1.0.1.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+* stop VM, the state of the VM changes to VM error, pause a vm, an inappropriate message will be displayed in the vm list [[#1987](https://github.com/harvester/harvester/issues/1987)]
+* VM created by VM Template with Access Credentials use the same secret [[#1989](https://github.com/harvester/harvester/issues/1989)]
+* Unable to join a cluster [[#1993](https://github.com/harvester/harvester/issues/1993)]
+* harvester-cloud-provider rely on CNI to communicate with host [[#1999](https://github.com/harvester/harvester/issues/1999)]
+* Harvester UI cloudconfig `packages` option being overwritten on VM creation [[#2006](https://github.com/harvester/harvester/issues/2006)]
+
+# Known Issues
+* Deleting a host holding the VIP in the cluster makes the VIP inaccessible [[#1398](https://github.com/harvester/harvester/issues/1398)]
+* Manual upgrade from 0.3.0 to 1.0.0 [[#1665](https://github.com/harvester/harvester/issues/1665)]
+* When using Load Balancer IPAM `Pool`s, user is only able to provision 1 LB successfully. Subsequent LBs using `Pool` will fail [[#2041](https://github.com/harvester/harvester/issues/2041)]
```
Comment (guangbochen) [inline v1.0.1.md:82]: Can u please check with David or Lanfo if they can reproduce it? at least I can't reproduce it in my env so might be an env related issue.

**Review by guangbochen — CHANGES_REQUESTED** [review]

File: v1.0.1.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Harvester v1.0.1 Release Note
+
+Harvester v1.0.1 is the second General Availability (GA) release of Harvester!
```
[the remark this answers is not in the document]
Comment (rebeccazzzz) [reply v1.0.1.md]: Changed line 3 to the suggested text.

File: v1.0.1.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* clone volume has an error attach vm [[#1977](https://github.com/harvester/harvester/issues/1977)]
+* stop VM, the state of the VM changes to VM error, pause a vm, an inappropriate message will be displayed in the vm list [[#1987](https://github.com/harvester/harvester/issues/1987)]
+* VM created by VM Template with Access Credentials use the same secret [[#1989](https://github.com/harvester/harvester/issues/1989)]
+* Unable to join a cluster [[#1993](https://github.com/harvester/harvester/issues/1993)]
+* harvester-cloud-provider rely on CNI to communicate with host [[#1999](https://github.com/harvester/harvester/issues/1999)]
+* Harvester UI cloudconfig `packages` option being overwritten on VM creation [[#2006](https://github.com/harvester/harvester/issues/2006)]
+
+# Known Issues
+* Deleting a host holding the VIP in the cluster makes the VIP inaccessible [[#1398](https://github.com/harvester/harvester/issues/1398)]
+* Manual upgrade from 0.3.0 to 1.0.0 [[#1665](https://github.com/harvester/harvester/issues/1665)]
```
[the remark this answers is not in the document]
Comment (rebeccazzzz) [reply v1.0.1.md]: Removed “manual upgrade from 0.3.0 to 1.0.0” from known issues.

File: v1.0.1.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+@nunix 
+@rebeccazzzz
+@TachunLin 
+@tjjh89017 
+@w13915984028 
+@weihanglo 
+@WuJun2016 
+@yaocw2020
+@yasker
+
```
[the remark this answers is not in the document]
Comment (rebeccazzzz) [reply v1.0.1.md]: Added suggested contributors.

File: v1.0.1.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+| KubeVirt | [v0.49.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.49.0) |
+| Rancher | [v2.6.4](https://github.com/rancher/rancher/releases/tags/v2.6.4) |
+| RKE2 | [v1.22.7+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.22.7%2Brke2r1) |
+
+# Knowledge Base Articles
+* [Multiple NICs VM Connectivity](https://harvesterhci.io/kb/multiple-nics-vm-connectivity/)
+* [VM Scheduling](https://harvesterhci.io/kb/vm-scheduling)
+
+# Contributors
+Thanks to all the contributors that make Harvester v1.0.1 GA possible!
```
Comment (guangbochen) [inline v1.0.1.md:102]:
```suggestion
Thanks to all the contributors that make the Harvester v1.0.1 release possible!
```

File: v1.0.1.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+## ISO
+:cd: [harvester-v1.0.1-amd64.iso](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-amd64.iso)
+
+## Additional Artifacts
+:file_folder: [harvester-v1.0.1-initrd-amd64](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-initrd-amd64)
+:file_folder: [harvester-v1.0.1-vmlinuz-amd64](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-vmlinuz-amd64)
+:file_folder: [harvester-v1.0.1-rootfs-amd64.squashfs](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-rootfs-amd64.squashfs)
+:white_check_mark: [harvester-v1.0.1-amd64.sha512](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-amd64.sha512)
+
```
Comment (guangbochen) [inline v1.0.1.md:17]:
```suggestion
:white_check_mark: [version.yaml](https://releases.rancher.com/harvester/v1.0.1/version.yaml)

```

File: v1.0.1.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+@ibrokethecloud 
+@joeyloman 
+@johnliu55tw 
+@joshmeranda 
+@lanfon72 
+@n313893254 
+@noahgildersleeve 
+@nunix 
+@rebeccazzzz
+@rosskirkpat
```
Comment (guangbochen) [inline v1.0.1.md:122]:
```suggestion
@rosskirkpat 
@slash2314
```

File: v1.0.1.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+:file_folder: [harvester-v1.0.1-rootfs-amd64.squashfs](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-rootfs-amd64.squashfs)
+:white_check_mark: [harvester-v1.0.1-amd64.sha512](https://releases.rancher.com/harvester/v1.0.1/harvester-v1.0.1-amd64.sha512)
+
+# Install
+Harvester can be installed with the 
+[ISO](https://docs.harvesterhci.io/v1.0/install/iso-install/) image or via
+[PXE](https://docs.harvesterhci.io/v1.0/install/pxe-boot-install/) boot.
+
+# Highlights
+* Upgrade from v1.0.0. [[#1861](https://github.com/harvester/harvester/issues/1861)]
```
Comment (guangbochen) [inline v1.0.1.md:24]:
```suggestion
* Upgrade from v1.0.0. [[#1861](https://github.com/harvester/harvester/issues/1861)] [[Doc](https://docs.harvesterhci.io/v1.0/upgrade/automatic/)]
```

File: v1.0.1.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+
+# Known Issues
+* Deleting a host holding the VIP in the cluster makes the VIP inaccessible [[#1398](https://github.com/harvester/harvester/issues/1398)]
+* Upgrade: rancher-monitoring charts can't be upgraded [[#1983](https://github.com/harvester/harvester/issues/1983)]
+* Rancher upgrade fail: Failed to find system chart "fleet" [[#2011](https://github.com/harvester/harvester/issues/2011)]
+* Upgrade: VMs fail to live-migrate to other hosts in some cases [[#2029](https://github.com/harvester/harvester/issues/2029)]
+* When using Load Balancer IPAM `Pool`s, user is only able to provision 1 LB successfully. Subsequent LBs using `Pool` will fail [[#2041](https://github.com/harvester/harvester/issues/2041)]
+* Upgrade: longhorn-system can't be evicted [[#2053](https://github.com/harvester/harvester/issues/2053)]
+* Predictable interface names deviate from previous versions [[#2073](https://github.com/harvester/harvester/issues/2073)]
+* IPAM deployment through rancher is not working -> `health checks` disabled by default [[#2077](https://github.com/harvester/harvester/issues/2077)]
```
Comment (guangbochen) [inline v1.0.1.md:87]:
```suggestion
* Deleting a host holding the VIP in the cluster makes the VIP inaccessible [[#1398](https://github.com/harvester/harvester/issues/1398)]

* When using Load Balancer IPAM `Pool`s, user is only able to provision 1 LB successfully. Subsequent LBs using `Pool` will fail [[#2041](https://github.com/harvester/harvester/issues/2041)]
* Upgrade: longhorn-system can't be evicted [[#2053](https://github.com/harvester/harvester/issues/2053)]
* Predictable interface names deviate from previous versions [[#2073](https://github.com/harvester/harvester/issues/2073)]
* Upgrade related:
  * rancher-monitoring charts can't be upgraded [[#1983](https://github.com/harvester/harvester/issues/1983)]
  * Rancher upgrade fail: Failed to find system chart "fleet" [[#2011](https://github.com/harvester/harvester/issues/2011)]
  * VMs fail to live-migrate to other hosts in some cases [[#2029](https://github.com/harvester/harvester/issues/2029)]
```

File: v1.0.1.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+| Component | Version |
+| ------ | ---------|
+| Longhorn | [v1.2.4](https://github.com/longhorn/longhorn/releases/tag/v1.2.4) |
+| KubeVirt | [v0.49.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.49.0) |
+| Rancher | [v2.6.4](https://github.com/rancher/rancher/releases/tags/v2.6.4) |
+| RKE2 | [v1.22.7+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.22.7%2Brke2r1) |
+
+# Knowledge Base Articles
+* [Multiple NICs VM Connectivity](https://harvesterhci.io/kb/multiple-nics-vm-connectivity/)
+* [VM Scheduling](https://harvesterhci.io/kb/vm-scheduling)
```
Comment (guangbochen) [inline v1.0.1.md:99]:
```suggestion
* [VM Scheduling](https://harvesterhci.io/kb/vm-scheduling)
* [NIC Naming Scheme changed after upgrading to v1.0.1](https://harvesterhci.io/kb/nic-naming-scheme)
```

File: v1.0.1.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+* Predictable interface names deviate from previous versions [[#2073](https://github.com/harvester/harvester/issues/2073)]
+* IPAM deployment through rancher is not working -> `health checks` disabled by default [[#2077](https://github.com/harvester/harvester/issues/2077)]
+
+# Dependency Component Versions
+| Component | Version |
+| ------ | ---------|
+| Longhorn | [v1.2.4](https://github.com/longhorn/longhorn/releases/tag/v1.2.4) |
+| KubeVirt | [v0.49.0](https://github.com/kubevirt/kubevirt/releases/tag/v0.49.0) |
+| Rancher | [v2.6.4](https://github.com/rancher/rancher/releases/tags/v2.6.4) |
+| RKE2 | [v1.22.7+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.22.7%2Brke2r1) |
```
Comment (guangbochen) [inline v1.0.1.md:95]:
```suggestion
| RKE2 | [v1.21.11+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.21.11%2Brke2r1) |
```

**Review by guangbochen — APPROVED** [review]

**Pull request merged**